src/routes/(admin)/admin/sliders/[[cursor]]/CreateSlider.svelte

```
@@ -0,0 +1,71 @@
+  });
+
+  let btnColor = $derived(slider.id ? 'bg-amber-700 hover:bg-amber-800 hover:text-neutral-50 focus:bg-amber-800 border-amber-600' : 'bg-teal-900 hover:bg-teal-800 hover:text-neutral-50 focus:bg-teal-800 border-teal-600/50')
+</script>
+
+<button
+  type="button"
+  class="inline-flex w-40 items-center justify-center gap-x-2 rounded-lg border border-transparent px-4 py-3 text-sm bg-brand-600 hover:bg-brand-700 focus:bg-brand-700 font-medium text-white focus:outline-none disabled:pointer-events-none disabled:opacity-50"
+  aria-haspopup="dialog"
+  aria-expanded="false"
+  aria-controls="slider-modal"
+  data-hs-overlay="#slider-modal" on:click={() => {slider = {id: undefined}}}>
```

ESLint

[error] Using `on:click` to listen to the click event is deprecated. Use the event attribute `onclick` instead
https://svelte.dev/e/event_directive_deprecated(event_directive_deprecated)

(svelte/valid-compile)